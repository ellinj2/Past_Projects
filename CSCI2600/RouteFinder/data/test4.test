r
200
201
r
201
201
r
98
99
r
86
91
r
76
76
q
