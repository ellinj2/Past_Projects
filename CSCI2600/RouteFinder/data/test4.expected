First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [200]
Unknown building: [201]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [201]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [98]
Unknown building: [99]
First building id/name, followed by Enter: Second building id/name, followed by Enter: There is no path from Polytechnic Residence Commons to OGE.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from EMPAC to EMPAC:
Total distance: 0.000 pixel units.
